stix_shifter/scripts/stix_shifter.py

```diff
@@ -6,12 +6,13 @@
 import json
 import time
 from stix_shifter_utils.utils.proxy_host import ProxyHost
+from stix_shifter_utils.utils.module_discovery import module_list, process_dialects
 
 TRANSLATE = 'translate'
 TRANSMIT = 'transmit'
 EXECUTE = 'execute'
 HOST = 'host'
-
+CONNECTOR_MODULES = module_list()
```

Comment:
does this need to be global? looks like only usage of this variable is to decide `help_and_exit` flag